streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -243,6 +254,22 @@ public class StreamsConfig extends AbstractConfig {
      */
     public static final String OPTIMIZE = "all";
 
+    /**
+     * Config value for parameter {@link #TOPOLOGY_OPTIMIZATION_CONFIG "topology.optimization"}
+     * for enabling the specific optimization that reuses source topic as changelog topic
+     * for KTables.
+     */
+    public static final String REUSE_KTABLE_SOURCE_TOPICS = "topology.optimization.reuse.ktable.source.topics";
```

Comment:
In the KIP, seems we do not have the `topology.optimization.` prefix for such values? Also I personally feel that we do not need those prefixes since for users who do want to list specific rules, they would become too long.